### ISSUE TITLE
Intercept address links? 

### ISSUE THREAD

**Issue by ProjectMoon** [gjedeer/Acastus#9]
Feature request: intercept address links, search for the address and then pass it off to the map app. This would primarily be to get around OsmAnd's terrible searching in an automated way.

**Comment by gjedeer** [comment]
On Wed, 16 Oct 2019 04:51:41 -0700
ProjectMoon <notifications@github.com> wrote:

> Feature request: intercept address links, search for the address and
> then pass it off to the map app. This would primarily be to get
> around OsmAnd's terrible searching in an automated way.
> 

Could you give me an example of such a link?

**Comment by gjedeer** [comment]
On Thu, 17 Oct 2019 05:02:13 -0700
ProjectMoon <notifications@github.com> wrote:

> I guess I mean binding to the map intent. One can tap addresses and
> they open in maps. Basically I want some kind of way to delegate
> OsmAnd to Acastus for searching. 
> 

Yeah, I understand what you want but I've never came across such a
link.

If you could send me an example of a website or an app where you've seen
a link like this, I could try implementing it.

**Comment by ProjectMoon** [comment]
I am not sure if it's actually a specific link or just Android turning addresses into links. It's what shows up in most calendar apps (e.g. Google Calendar, Etar).

**Comment by gjedeer** [comment]
Okay, that's something I can reproduce. 

```
intent:0,0?q=Słup Szymon, galeria plakatu#Intent;scheme=geo;launchFlags=0x3000000;S.com.android.browser.application_id=com.android.calendar;end 
------------ 
ACTION: android.intent.action.VIEW 
DATA: geo:0,0?q=Słup Szymon, galeria plakatu 
URI: intent:0,0?q=Słup Szymon, galeria plakatu#Intent;scheme=geo;launchFlags=0x3000000;S.com.android.browser.application_id=com.android.calendar;end 
FLAGS: 
FLAG_ACTIVITY_FORWARD_RESULT 
FLAG_ACTIVITY_PREVIOUS_IS_TOP 
EXTRAS: 
1 Class: java.lang.String 
Key: com.android.browser.application_id 
Value: com.android.calendar 

------------ 
MATCHING ACTIVITIES: 
Vespucci (de.blau.android - de.blau.android.GeoUrlActivity) 
Share on OpenStreetMap (de.blau.android - de.blau.android.ShareOnOpenStreetMap) 
Transportr Devel (de.grobox.liberario.debug - de.grobox.transportr.map.MapActivity) 
A Photo Map (de.k3b.android.androFotoFinder - de.k3b.android.androFotoFinder.locationmap.MapGeoPickerActivity) 
Offi Directions (de.schildbach.oeffi - de.schildbach.oeffi.directions.DirectionsActivity) 
Street­Complete (de.westnordost.streetcomplete - de.westnordost.streetcomplete.MainActivity) 
OsmAnd~ (net.osmand.plus - net.osmand.plus.activities.search.GeoIntentActivity) 

```

**Comment by k3b** [comment]
Implemented with #16
